cell_classification/prepare_data_script.py

```diff
@@ -25,7 +25,7 @@ def naming_convention(fname):
     normalization_dict_path=os.path.normpath(
        "C:/Users/lorenz/Desktop/angelo_lab/TONIC/normalization_dict.json"
     ),
-    normalization_quantile=0.99,
+    normalization_quantile=0.999,
```

Comment:
Will you plot a couple side-by-side, along with the raw image scaled 0-1 to max, to see what the effects are?

Reply:
Will do before I create the decidua dataset.